### ISSUE TITLE
Wandb bug when running train long

### ISSUE THREAD

**Issue by isaacmg** [AIStream-Peelout/flow-forecast#35]
When running this code https://github.com/AIStream-Peelout/flow-forecast/commit/1f67ac4844859e5d60a0f5dba2dbbe8f4c5dbc30 from a colab notebook Wandb views the entire thing as one training session and continue gradient steps indefinitely. Training session should be forced to end when that model stops training not when the meta training loop finishes. Should only be 28 training steps not 80.
<img width="1094" alt="image" src="https://user-images.githubusercontent.com/3865062/71710653-65567f80-2dcb-11ea-8558-0f3280c4ab7b.png">
